doc/workspace_symbol_params.md

```diff
@@ -36,6 +36,7 @@ We extend the `WorkspaceSymbolParams` by adding extra fields:
   - whole_word to match the whole word instead of a part of it when True
   - negate to invert matching
   - kind is a search kind described above
+  - uri returns entities only for this file
```

Comment:
In my opinion URI shouldn't be present in Workspace Symbol request. Use Document Symbol request instead. 

I approve this as a temporary solution, but if you merge the PR I'll open a TN to fix this latter.

Reply:
I've already prepared a new patch without this "uri" and in the future Document Symbol request will be expanded to accept filtering options